src/Mantasflowers.Domain/Entities/UserAddress.cs

```diff
@@ -1,4 +1,11 @@
+using System;
+
 namespace Mantasflowers.Domain.Entities
 {
-    public class UserAddress : Address {}
+    public class UserAddress : Address 
+    {
+        public Guid UserId { get; set; }
+
+        public virtual User User { get; set; }
+    }
```

Comment:
It might be worth a shot to do the same with OrderAddress, considering that it might be volatile to this change too. For example, a case when a courier would only require delivery address to display.